xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -20,6 +20,35 @@
 #include "utils/log.h"
 #include "windowing/GraphicContext.h"
 
+namespace
+{
+void AppendFont(const std::string font)
+{
+  std::string finalFontPath = URIUtils::AddFileToFolder("special://temp/fonts/", font);
+  if (XFILE::CFile::Exists(finalFontPath))
+  {
+    CLog::Log(LOGDEBUG,
+              "CDVDSubtitlesLibass: Skipping copy of {} to special://temp/fonts/ (already exists)",
+              font);
+    return;
+  }
+
+  std::string fontSources[]{"special://home/media/Fonts/", "special://xbmc/media/Fonts/"};
+
+  for (const auto& path : fontSources)
+  {
+    auto fontPath = URIUtils::AddFileToFolder(path, font);
+    if (XFILE::CFile::Exists(fontPath))
+    {
+      XFILE::CFile::Copy(fontPath, finalFontPath);
+      CLog::Log(LOGDEBUG, "CDVDSubtitlesLibass: Copied {} to {}", fontPath, finalFontPath);
+      return;
```

Comment:
Should you return earlier here? Will the the second path exist if the first path does?

Reply:
Second folder (system one) is only used if the font being added can not be found in the media/fonts folder. It was implemented like this before as well. 
As far as I can see, kodi will include arial and teletext by default in the system folder. You should be able to select those fonts and use the setting even if you have not added any fonts to your media folder.